### PR TITLE
Fix buttons height on mobile on order view page

### LINKED ISSUE #22660: [RESPONSIVE - BO] Order page buttons are taking too much height on mobile
#### Describe the bug

Order page buttons are taking too much height on mobile

#### Expected behavior

image

#### Steps to Reproduce

1. Go to Order page

**Screenshots**

image

#### Additional information

* PrestaShop version: N/A
* PHP version: N/A

### PR DESCRIPTION
**Pull request by NeOMakinG**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Buttons were strange on 375px, now it's correctly aligned
| Type?             | bug fix
| Category?         | BO
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #22660.
| How to test?      | Go on order view page and see on 375px

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/22662)

### CODE REVIEW

**Review by matthieu-rolland — DISMISSED** [review]

File: admin-dev/themes/new-theme/scss/pages/orders/_orders_view.scss [hunk 1]
```diff
@@ -439,6 +424,21 @@
...
+        width: 100%;
+        max-width: 242px;
+
+        > .input-group,
+        button {
+          width: 100%;
+          text-align: center;
+
+          i {
+            margin-right: .5rem;
```
Comment (atomiix) [inline admin-dev/themes/new-theme/scss/pages/orders/_orders_view.scss]:
```suggestion
            margin-right: 0.5rem;
```

File: admin-dev/themes/new-theme/scss/pages/partials/_order-navigation.scss [hunk 2]
```diff
@@ -17,6 +17,9 @@
   #order-view-page {
     .order-navigation {
       justify-content: space-between;
+      width: 100%;
+      max-width: 88px;
+      margin-left: .8rem;
```
Comment (atomiix) [inline admin-dev/themes/new-theme/scss/pages/partials/_order-navigation.scss]:
```suggestion
      margin-left: 0.8rem;
```

File: admin-dev/themes/new-theme/scss/pages/orders/_orders_view.scss (hunk 1 above)
Comment (NeOMakinG) [reply admin-dev/themes/new-theme/scss/pages/orders/_orders_view.scss]: We've a lint rule for this, it's not related to this PR at all that shouldn't be blocking

File: admin-dev/themes/new-theme/scss/pages/partials/_order-navigation.scss (hunk 2 above)
Comment (NeOMakinG) [reply admin-dev/themes/new-theme/scss/pages/partials/_order-navigation.scss]: Same as above

File: admin-dev/themes/new-theme/scss/pages/orders/_orders_view.scss (hunk 1 above)
Comment (NeOMakinG) [reply admin-dev/themes/new-theme/scss/pages/orders/_orders_view.scss]: For reference : https://github.com/PrestaShop/PrestaShop/pull/22670

**Review by atomiix — DISMISSED** [review]

**Review by PierreRambaud — DISMISSED** [review]

**Comment by Roomsmaniia** [comment]
Hi @NeOMakinG,
I saw in the PR the change of the label on the button "Update status" I change it back to WFdev

**Review by Progi1984 — DISMISSED** [review]

**Comment by NeOMakinG** [comment]
@RomainBocheux We decided to keep the label as it is, as it can become harder to maintain

**Comment by SD1982** [comment]
Hi @NeOMakinG with your PR there is a large space above the buttons now
Is this intended behavior?
**Without PR**
image

**With PR**
image

Tested with Chrome

**Comment by NeOMakinG** [comment]
@SD1982 did you try to reload the page on mobile ? I think there is something with JS on the whole BO where you need to go on the page on mobile, but you can't resize without reloading the page overwise the padding is too big

**Comment by SD1982** [comment]
Hi @NeOMakinG I remove the "waiting for author" label i will test a second time :+1:

**Comment by SD1982** [comment]
Hi @NeOMakinG after a retest with browser dev tools it's ok after a refresh 
image

But if i test with browserstack it look like this
**BO in english**
image
**BO in french**
image

Emulated device => Samsung S20 with Chrome
WDYT ?

**Comment by NeOMakinG** [comment]
@SD1982 I adjusted a bit on very small mobile, your emulator looks to be under 375px!

**Review by Progi1984 — DISMISSED** [review]

**Comment by Progi1984** [comment]
@NeOMakinG Lint CSS is :red_circle:

**Review by atomiix — APPROVED** [review]

**Review by Progi1984 — APPROVED** [review]

**Comment by SD1982** [comment]
Hi @NeOMakinG 
Looks good now in browser dev tools or browserstack
image
image

Tested on Chrome and Firefox
Thanks @NeOMakinG !!

**Comment by Progi1984** [comment]
Thanks @NeOMakinG & @SD1982

**Pull request merged**